testfiles/Attic_ASTM_C1340_Base_Case.idf

```diff
@@ -126,123 +151,194 @@
     Yes,                     !- Use Weather File Rain Indicators
     Yes;                     !- Use Weather File Snow Indicators
 
-!-   ===========  ALL OBJECTS IN CLASS: GROUNDTEMPERATURES ===========
 
-  Site:GroundTemperature:BuildingSurface,18.3,18.2,18.3,18.4,20.1,22.0,22.3,22.5,22.5,20.7,18.9,18.5;
+!-   ===========  ALL OBJECTS IN CLASS: SITE:GROUNDTEMPERATURE:BUILDINGSURFACE ===========
+
+Site:GroundTemperature:BuildingSurface,
+    23.0,                    !- January Ground Temperature {C}
```

Comment:
@mitchute Did you intend to change the ground temps here?  I guess looking further down this is more than just a transition/cleanup.